addons/l10n_es_pos/models/pos_config.py

```diff
@@ -0,0 +1,27 @@
+from odoo import _, api, fields, models
+
+
+class PosConfig(models.Model):
+    _inherit = "pos.config"
+
+    is_spanish = fields.Boolean(string="Company located in Spain", compute="_is_company_spanish")
+
+    def _is_company_spanish(self):
+        for pos in self:
+            pos.is_spanish = pos.company_id.country_id.code == "ES"
+
+    l10n_es_simplified_invoice_limit = fields.Float(
+        string="Simplified Invoice limit amount",
+        digits="2",
```

Comment:
```suggestion
        digits=2,
```

use int, not string.